api/v2/api_test.go

```diff
@@ -663,7 +664,7 @@ func TestAPI_HandleQueuError_Failure(t *testing.T) {
 	fakeLens := &mocks.FakeIndexerAPIClient{}
 	fakeOrch := &mocks.FakeServiceClient{}
 	fakeSigner := &mocks.FakeSignerClient{}
-	api, err := Initialize(cfg, "", true, logger, fakeLens, fakeOrch, fakeSigner)
+	api, err := Initialize(cfg, "", true, true, logger, fakeLens, fakeOrch, fakeSigner)
```

Comment:
why do we have `debug` AND `devMode` as params? to me they imply the same thing

if we need it, I would rather a option struct be introduced:

```go
type Options struct {
   DebugLogs bool
   DevMode bool
}
```